teamcity/project_test.go

```diff
@@ -66,6 +66,27 @@ func TestProject_UpdateWithSameParentDoesNotChangeName(t *testing.T) {
 	assert.Equal(t, "ChildProject", updated.Name)
 }
 
+func TestProject_UpdateName(t *testing.T) {
```

Comment:
I believe for this test, there's no need to create both `parent` and `child` projects

Reply:
👍 pushed